theme/static/css/hydroshare_core.css

```diff
@@ -3665,6 +3665,9 @@ input.no-style {
 div.group-thumbnails {
 	display: flex;
```

Comment:
We can remove the `display: flex` and `flex-wrap` rules here because they do not apply to the grid. 

Also, we can simplify 
`grid-template-columns: repeat(auto-fill, minmax(min(33.3rem, 100%), 1fr));` 
to 
`grid-template-columns: repeat(auto-fill, minmax(33.3rem, 1fr));`

Then add to the items
`flex-grow: 1` to achieve a similar result.

Reply:
thanks @Maurier I didn't know you could use flex-grow outside of a flex container...